### PR TITLE
feature: added Dockerfile example

### PR DESCRIPTION
**Pull request by erwanvivien**
I believe Docker only could be useful too :)

### CODE REVIEW

**Comment by codetheweb** [comment]
Sorry, I'm a little confused what this is for.

Do you mean it would be useful to have a one-line `docker run ...` command in addition to the Docker Compose example?

**Comment by erwanvivien** [comment]
I'm sorry, I explained myself badly. I don't personally use Docker-compose, so I made the docker Compose as a Dockerfile to be able to run it as a simple Docker

**Comment by codetheweb** [comment]
Got it, so would a `docker run ...` example make more sense here then?

**Comment by erwanvivien** [comment]
I believe so ! :)

**Comment by sibartel** [comment]
I agree, creating a new docker image just to set env vars seems a bit over the top.

@erwanvivien, maybe edit your proposed readme section to contain info about how to invoke `docker run` with the right env vars set ([docker doc](https://docs.docker.com/engine/reference/commandline/run/#set-environment-variables--e---env---env-file))

Maybe also about the use of an env file.

**Comment by erwanvivien** [comment]
Fixed and updated in 7363191093441c3e36fe5622c5ae6ab573d69196

File: README.md [hunk 1]
```diff
@@ -35,6 +35,19 @@ services:
       - DISCORD_USER_ID= # Discord user ID of the user you want Aoede to follow
 ```
 
+**Docker**:
+```env
+# .env
+ENV DISCORD_TOKEN=
```
Comment (sibartel) [inline README.md:41]: here should be no `ENV` just `DISCORD_TOKEN=XY`, the same for the lines below

File: README.md [hunk 2]
```diff
@@ -35,6 +35,19 @@ services:
...
+ENV SPOTIFY_USERNAME=
+ENV SPOTIFY_PASSWORD=
+ENV DISCORD_USER_ID=
+```
+
+```bash
+docker run -d --env-file .env codetheweb/aoede
```
Comment (sibartel) [inline README.md:48]: Maybe add `--rm` to automatically remove the container? Don't know

File: README.md (hunk 1 above)
Comment (erwanvivien) [reply README.md:41]: My bad...

File: README.md (hunk 2 above)
Comment (erwanvivien) [reply README.md:48]: Sure, will do :)

File: README.md (hunk 1 above)
Comment (erwanvivien) [reply README.md]: Fixed in 929c989798a09cd47f5a690944d22443ecf92f44

File: README.md (hunk 2 above)
Comment (erwanvivien) [reply README.md]: Fixed in 929c989798a09cd47f5a690944d22443ecf92f44

**Comment by codetheweb** [comment]
Thanks @erwanvivien and @sibartel. :)

**Pull request merged**